bootstrap/absolute.sh

```diff
@@ -17,7 +17,10 @@ set_path_env $(bootstrap_command_path)
 set_path_env $(absolute_path)/node_modules/.bin
 
 # NPM update
-npm update
+if [ ! .pkg_timestamp -nt package.json ]; then
```

Comment:
Oh, I agree with your opinion.
BTW, I did try what you said but it works well in my environment. (Mac)
Which platform did you try it on?